redbot/__main__.py

```diff
@@ -116,8 +116,9 @@ def main():
     red.add_cog(CogManagerUI())
     if cli_flags.dev:
         red.add_cog(Dev())
+    loop = asyncio.get_event_loop()
```

Comment:
Already assigned in line 110, no need to refetch the loop here.

Reply:
Oops! Thanks, that one on line 110 was added in a merge from V3/develop